lib/linguist/languages.yml

```diff
@@ -5473,6 +5473,18 @@ Vim script:
   - vimrc
   ace_mode: text
   language_id: 388
+Vim snippet:
+  type: markup
+  aliases:
+  - SnipMate
+  - UltiSnip
```

Comment:
According to [its repository](https://github.com/SirVer/ultisnips), this engine's name should be `UltiSnips` (pluralised).

Keep the singular spelling too, though, because it's easily confused.